src/components/SourceEditForm/SourceEditModal.js

```diff
@@ -66,13 +66,13 @@ const SourceEditModal = () => {
     if (isLoading) {
         return (
             <Modal
-                title={intl.formatMessage({
+                aria-label={intl.formatMessage({
```

Comment:
Is the title no longer required?

Reply:
Yep. Do not use title when there is a custom header. Before that, the title was just used to generate aria-label, so now you have to pass it directly.

Reply:
Ah custom header :+1: